java/code/src/com/suse/manager/webui/controllers/CVEAuditController.java

```diff
@@ -141,15 +141,13 @@ public static Object cveAudit(Request req, Response res, User user) {
             switch (cveAuditRequest.getTarget()) {
                 case SERVER:
                     Set<Long> systemSet = RhnSetDecl.SYSTEMS.get(user).getElementValues();
-                    // TODO: Use CVEAuditManagerOVAL once it's ready
-                    List<CVEAuditServer> cveAuditServers = CVEAuditManager
+                    List<CVEAuditServer> cveAuditServers = CVEAuditManagerOVAL
                     .listSystemsByPatchStatus(user, cveAuditRequest.cveIdentifier,
                             cveAuditRequest.statuses);
                     cveAuditServers.forEach(serv -> serv.setSelected(systemSet.contains(serv.getId())));
                     return json(res, ResultJson.success(cveAuditServers));
                 case IMAGE:
-                    // TODO: Use CVEAuditManagerOVAL once it's ready
-                    List<CVEAuditImage> cveAuditImages = CVEAuditManager
+                    List<CVEAuditImage> cveAuditImages = CVEAuditManagerOVAL
                     .listImagesByPatchStatus(user, cveAuditRequest.cveIdentifier,
                             cveAuditRequest.statuses);
```

Comment:
The reason is two folds: 
1. Currently it's not clear how images would be audited using OVAL data, primarily because it's not possible to extract the CPE (or the operating system) of the image without deploying it. @cbosdo suggested that we can derive the CPE from the channels assigned to the image, but I haven't had time to experiment with the feasibility of that option and what edge cases we might encounter, so the best decision I saw, was to keep using channels for auditing images temporarily.
2. The reason for having `listImagesByPatchStatus` even though it is just delegating to `CVEAuditManager` is just a matter of taste; having a single source of truth where all the CVE auditing logic would reside seemed reasonable from an architectural pov.

Reply:
@HoussemNasri can you please create an issue that Oval based scanning will be limited to systems only and not to images and this will be TODO that one might pickup in the future. Maybe also add the comment in this method about this , that @wweellddeerr has pointed out.

Reply:
I created an item in the project board: https://github.com/orgs/uyuni-project/projects/3/views/1?pane=issue&itemId=63725688. Will transfer into a real issue once PR is merged.